tests/categorical_test.py

```diff
@@ -1,9 +1,15 @@
 import numpy as np
+import os
 import tempfile
+import io_util
 from context import arkouda as ak
 from base_test import ArkoudaTest
 
 
+cat_test_base_tmp = '{}/categorical_test'.format(os.getcwd())
+io_util.get_directory(cat_test_base_tmp)
```

Comment:
I think the main thing `get_directory` does is create a path if it doesn't exist, and `TemporaryDirectory` should do that already so I'm not sure if this is needed.

Reply:
It bombed on me when I didn't call `get_directory`.  It's really because the `TemporaryDirectory` wrapper is creating another directory under it.  However, now that I look at it, it doesn't clean up the sub dir like I expected it to.  I'm going to postpone this one for a bit until I can get a cleaner approach.